cirq/study/sweepable.py

```diff
@@ -35,20 +35,8 @@
 
 def to_resolvers(sweepable: Sweepable) -> Iterator[ParamResolver]:
     """Convert a Sweepable to a list of ParamResolvers."""
-    if sweepable is None:
-        yield ParamResolver({})
-    elif isinstance(sweepable, ParamResolver):
-        yield sweepable
-    elif isinstance(sweepable, Sweep):
-        yield from sweepable
-    elif isinstance(sweepable, dict):
-        yield ParamResolver(cast(Dict, sweepable))
```

Comment:
I would like to note that this change is not exactly equivalent: 

This is passing with the new version: 
```
    assert [cirq.ParamResolver({'a': 2, 'b':1})] == list(cirq.to_resolvers({'a':[2], 'b': [1]}))
```
but fails in master: 

```
E       AssertionError: assert [cirq.ParamResolver({'a': 2, 'b': 1})] == [cirq.ParamResolver({'a': [2], 'b': [1]})]
```
However, I don't think this is a big issue. We do not officially support list values here, so I don't think our users used array values at all. I saw one example in https://github.com/quantumlib/Cirq/issues/3196 - but I can't figure out how to resolve that array value there anyway.